File: app/models/review.rb
class Review < ApplicationRecord
  validates :body, presence: true
  validates :rating, numericality: true, :inclusion => 1..100
  validates :votes, numericality: true

  belongs_to :user
  belongs_to :release
  has_many :upvotes
  has_many :downvotes



end

File: react/src/components/Review.js
import React from 'react';

const Review = (props) => {
  return(
    <div className="review">
    <div>Review body:{props.body}</div>
    <br />
    <div>Review rating:{props.rating}</div>
    <div>Review votes:{props.votes}</div>
    </div>
  )
}

export default Review;
